cmd/pmm-managed/main.go

```diff
@@ -97,6 +97,8 @@ var (
 	agentRDSExporterConfigF = flag.String("agent-rds-exporter-config", "/etc/percona-rds-exporter.yml", "rds_exporter configuration file path")
 	agentQANBaseF           = flag.String("agent-qan-base", "/usr/local/percona/qan-agent", "qan-agent installation base path")
 
+	rdsEnableGovCloud = flag.Bool("rds-enable-gov-cloud", false, "Enable GOV cloud for RDS")
```

Comment:
Is there a downside of always trying govcloud?

Reply:
Let's change the default value to `true`, but keep the flag for now. It will make testing this feature by the customer easier, but allow us to change it later. I will ask @michaelcoburn what to do there.

Reply:
Yes, request returns 403 error.
```InvalidClientTokenId: The security token included in the request is invalid.\n\tstatus code: 403```

Reply:
I prepared feature build for customer to test it. https://github.com/Percona-Lab/pmm-submodules/pull/148